app/frontend/components/taskList/taskList.tsx

```diff
@@ -7,7 +7,7 @@ import { ErrorMessage } from '@hookform/error-message'
 import { DeleteTaskModal } from '../deleteTaskModal'
 import { useState } from 'react'
 import { yupResolver } from '@hookform/resolvers/yup'
-import { TaskDetailsModal, taskInputSchema } from '../../../frontend/components/taskDetailsModal'
+import { taskInputSchema } from '../../../frontend/components/taskDetailsModal'
```

Comment:
Can we delete the `TaskDetailsModal` component, or is it still used elsewhere?

Reply:
yes, we use ```yup resolver``` for validation field

Reply:
Okay, then I would say let's put the validation in this file and delete the other one.
We don't need any unused code.